aws/resource_aws_iam_role_policy_test.go

```diff
@@ -479,13 +479,15 @@ resource "aws_iam_role_policy" "foo" {
 	name = "tf_test_policy_%s"
 	role = "${aws_iam_role.role.name}"
 	policy = <<EOF
-  {
+  x{
```

Comment:
Is this deliberate?

Reply:
Yes.  That resource is intended to trigger the "invalid JSON" path.  Before this patch, it was triggered by the indentation, which is now considered valid.  In retrospect, instead of "x" I should have put something ilke "this_json_is_intentionally_invalid_for_testing".

I haven't touched this code in ~6 months but I will try to find time this week to rebase, verify tests still pass, and make this intent clearer.